test/sdk/sdk.test.ts

```diff
@@ -0,0 +1,324 @@
+import _ from 'lodash';
+import BN from 'bn.js';
+import moment from 'moment';
+import type { KeyringPair } from '@polkadot/keyring/types';
+import { u8aToHex } from '@polkadot/util';
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import { Chain, ChainAsset, Weight }  from '@oak-network/sdk-types';
+import { assets, chains } from '@oak-network/config';
+import { MoonbeamAdapter, MangataAdapter, AstarAdapter, OakAdapter } from '@oak-network/adapter';
+import { Sdk } from '@oak-network/sdk';
+import { rpc, types, runtime } from '@oak-network/types';
+import { Mangata } from '@mangata-finance/sdk';
+import { getKeyringPair, getMoonbeamKeyringPair } from '../utils/helpFn';
+
+const getOakConfig = (): Chain => {
+  return process.env.ENV === 'Turing Dev' ? chains.turingLocal : chains.turingLocal;
+}
+
+const getOakConfigWithMoonbase = (): Chain => {
+  return process.env.ENV === 'Turing Dev' ? chains.turingLocal : chains.turingMoonbase;
+}
+
+const getMangataConfig = (): Chain => {
+  return process.env.ENV === 'Turing Dev' ? chains.mangataKusama : chains.mangataRococo;
+}
+
+const getAstarConfig = (): Chain => {
+  return process.env.ENV === 'Turing Dev' ? chains.shibuya : chains.rocstar;
+}
+
+const getMoonbeamConfig = (): Chain => {
+  return process.env.ENV === 'Turing Dev' ? chains.moonbaseLocal : chains.moonbaseAlpha;
+}
```

Comment:
These functions are so poorly written! 👎

You only need for one find function. Can’t chain be find by its key with a `_.find` lodash?
```
const getChainConfigOrDefault = (chainKey) => {
   ...
}
```

Reply:
Fixed